client/client.go

```diff
@@ -454,3 +454,18 @@ func (client *Client) CreateUsers(options []*CreateUserOptions) ([]*CreateUserRe
 	}
 	return results, nil
 }
+
+type DeviceUser struct {
```

Comment:
We need `ID int` here, as that's our internal key and the only thing we really need for a user to exist.